editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/utils/NodeUtils.java

```diff
@@ -153,5 +166,53 @@ public static AbstractCamelModelElement getSelectedNode(ISelection selection) {
 			answer = toCamelElement(input);
 		}
 		return answer;
+	}
+    
+    /**
+     * returns true if both nodes have the same parent
+     * 
+     * @param nodeA
+     * @param nodeB
+     * @return
+     */
+    public static boolean haveSameParent(AbstractCamelModelElement nodeA, AbstractCamelModelElement nodeB) {
+    	return nodeA.getParent().equals(nodeB.getParent());
```

Comment:
shouldn't a method added to AbstractCamelModelModelElement? Seems that it is a behavior related to AbstractCamelModelement only
AbstractCamelModelElement.haveSameParent(AbstractCamelModelElement node)
and then call
nodeB.haveSameParent(nodeA) ?